drakrun/drakrun/main.py

```diff
@@ -798,17 +792,14 @@ def validate_xen_commandline(ignore_failure: bool) -> None:
         "altp2m": "1",
         "hpet": "legacy-replacement",
     }
-
-    parsed_xl_info = get_xl_info()
-    xen_cmdline = get_xen_commandline(parsed_xl_info)
+    xen_info = get_xen_info()
+    xen_cmdline = parse_xen_commandline(xen_info["xen_commandline"])
```

Comment:
hm, to już chyba xen_info po prostu? bo z tego co widzę nie ma tego klucza już

Reply:
Ten klucz jest w ramach wartości z xen_info